easybuild/framework/easyblock.py

```diff
@@ -1566,6 +1570,24 @@ def sanity_check_step(self, custom_paths=None, custom_commands=None, extension=F
         else:
             self.log.info("Using specified sanity check paths: %s" % paths)
 
+            # transform specified libs into simple file paths
+            if 'libs' in paths.keys():
+                for lib in paths['libs']:
+                    if 'name' in lib.keys() and 'kind' in lib.keys():
+                        if lib['kind'] in LIB_SUFFIXES.keys():
+                            # TODO: also check possible library version numbers (e.g. libawesome.so.3.0.1)
+                            libpaths = ()
+                            for dir in LIB_DIRS:
+                                for suffix in LIB_SUFFIXES[lib['kind']]:
+                                    libpaths += (os.path.join(dir, lib['name'] + suffix), )
```

Comment:
use a list for `libpaths`, which is easier to update, and then use `tuple(libpaths)` below